[x86/Linux] No lldb package in x86 rootfs

While cross building for x86, I've got a CMake error::
`sudo ./cross/build-rootfs.sh x86 xenial lldb3.8`
`./build.sh x86 debug cross clang3.8 skipnuget`
```
CMake Error at src/ToolBox/SOS/lldbplugin/CMakeLists.txt:59 (message):
  Cannot find lldb-3.5, lldb-3.6, lldb-3.8 or lldb-3.9.  Try installing
  lldb-3.6-dev (or the appropriate package for your platform)


-- Configuring incomplete, errors occurred!
See also "/home/ivan/git/coreclr/bin/obj/Linux.x86.Debug/CMakeFiles/CMakeOutput.log".
~/git/coreclr
Failed to generate CoreCLR component build project!
```
Seems like lldb package added only for arm and armel in ./cross/build-rootfs.sh

```
if [[ "$__BuildArch" == "arm" ]]; then
    __UbuntuPackages+=" ${__LLDB_Package:-}"
fi

if [ "$__BuildArch" == "armel" ]; then
    __LLDB_Package="lldb-3.5-dev"
    __UbuntuPackages+=" ${__LLDB_Package:-}"
fi
```
It happens after commit daeb6927dc006fcda24aa3c7384f57da18a95191
\CC: @gkhanna79 @hqueue

--- a/cross/build-rootfs.sh
+++ b/cross/build-rootfs.sh
@@ -108,14 +108,10 @@ for i in "$@" ; do
     esac
 done
 
-if [[ "$__BuildArch" == "arm" ]]; then
-    __UbuntuPackages+=" ${__LLDB_Package:-}"
-fi
-
 if [ "$__BuildArch" == "armel" ]; then
     __LLDB_Package="lldb-3.5-dev"
-    __UbuntuPackages+=" ${__LLDB_Package:-}"
 fi
+__UbuntuPackages+=" ${__LLDB_Package:-}"
 
 __RootfsDir="$__CrossDir/rootfs/$__BuildArch"
 

